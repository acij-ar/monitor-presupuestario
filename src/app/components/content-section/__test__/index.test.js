const React = require('react');
const { shallow } = require('enzyme');

const ContentSection = require('..');

describe('ContentSection component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <ContentSection title="mocked title">
        Some content
      </ContentSection>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
