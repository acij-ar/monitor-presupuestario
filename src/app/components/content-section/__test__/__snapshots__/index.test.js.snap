// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentSection component should match snapshot 1`] = `
<div
  className="content-section"
>
  <h1
    className="content-section-title"
  >
    mocked title
  </h1>
  <div
    className="content-section-content"
  >
    Some content
  </div>
</div>
`;
